python/tests/test_metrics.py

```diff
@@ -309,34 +329,21 @@ def test_seldon_metrics_predict(cls):
 
     rv = client.get('/predict?json={"data": {"names": ["input"], "ndarray": ["data"]}}')
     assert rv.status_code == 200
+    assert ("metrics" in json.loads(rv.data)["meta"]) == client_gets_metrics
 
     data = seldon_metrics.data[os.getpid()]
-    assert data["GAUGE", "mygauge"]["value"] == 100
-    assert data["GAUGE", "customtag"]["value"] == 200
-    assert data["GAUGE", "customtag"]["tags"] == {"mytag": "mytagvalue"}
-    assert data["COUNTER", "mycounter"]["value"] == 1
-    assert np.allclose(
-        np.histogram([20.2 / 1000], BINS)[0], data["TIMER", "mytimer"]["value"][0]
-    )
-    assert np.allclose(data["TIMER", "mytimer"]["value"][1], 0.0202)
+    verify_seldon_metrics(data, 1, [0.0202])
 
     rv = client.get('/predict?json={"data": {"names": ["input"], "ndarray": ["data"]}}')
     assert rv.status_code == 200
+    assert ("metrics" in json.loads(rv.data)["meta"]) == client_gets_metrics
 
     data = seldon_metrics.data[os.getpid()]
-    assert data["GAUGE", "mygauge"]["value"] == 100
-    assert data["GAUGE", "customtag"]["value"] == 200
-    assert data["GAUGE", "customtag"]["tags"] == {"mytag": "mytagvalue"}
-    assert data["COUNTER", "mycounter"]["value"] == 2
-    assert np.allclose(
-        np.histogram([20.2 / 1000, 20.2 / 1000], BINS)[0],
-        data["TIMER", "mytimer"]["value"][0],
-    )
-    assert np.allclose(data["TIMER", "mytimer"]["value"][1], 0.0404)
+    verify_seldon_metrics(data, 2, [0.0202, 0.0202])
```

Comment:
Nice one adding the `verify_seldon_metrics()` helper! It makes the test way cleaner.